CONTRIBUTING.md

```diff
@@ -0,0 +1,316 @@
+We recommend watching the CI infrastructure throughout the next day to look for regressions.
+The CI infrastructure will directly email you if a group of commits that included yours caused a failure.
+You are expected to check those messages to see whether they are your fault and, if so, fix the breakage.
+
+Commits that clearly violate these quality standards may be reverted, in particular when the change blocks other developers from making progress.
+The developer is welcome to recommit the change after the problem has been fixed.
+
+### Release Branch Pull Requests
+
+A pull request targeting a release branch (`release/x.y` or `swift/release/x.y`) cannot be merged without a GitHub approval by a corresponding branch manager.
+In order for a change to be considered for inclusion in a release branch, the pull request must have:
+
+A title starting with a designation containing the release version number of the target branch.
+
+[This form](https://github.com/swiftlang/.github/blob/main/PULL_REQUEST_TEMPLATE/release.md?plain=1) filled out in its description.
+An item that is not applicable may be left blank or completed with an indication thereof, but must not be omitted altogether.
+
+To switch to this template when drafting a pull request in a [swiftlang](https://github.com/swiftlang) repository in a browser, append the `template=release.md` query parameter to the current URL and refresh. For example:
+
+```diff
+-https://github.com/swiftlang/swift/compare/main...my-branch?quick_pull=1
++https://github.com/swiftlang/swift/compare/main...my-branch?quick_pull=1&template=release.md
+```
+
+[Here](https://github.com/swiftlang/swift/pull/73697) is an example.
+
+### Commit Access
+
+Commit access is granted to contributors with a track record of submitting high-quality changes.
+If you would like commit access, please send an email to [the code owners list](code-owners@forums.swift.org) with the GitHub user name that you want to use and a list of 5 non-trivial pull requests that were accepted without modifications.
+
+Once you’ve been granted commit access, you will be able to commit to all of the GitHub repositories that host Swift.org projects.
+To verify that your commit access works, please make a test commit (for example, change a comment or add a blank line). The following policies apply to users with commit access:
+
+- You are granted commit-after-approval to all parts of Swift.
+To get approval, create a pull request. When the pull request is approved, you may merge it yourself.
+
+- You may commit an obvious change without first getting approval.
+The community expects you to use good judgment.
+Examples are reverting obviously broken patches, correcting code comments, and other minor changes.
+
+- You are allowed to commit changes without approval to the portions of Swift to which you have contributed or for which you have been assigned responsibility.
+Such commits must not break the build.
```

Comment:
CI testing prevents you from breaking the build. Let’s remove this.

Reply:
Should be gone now with the latest commit